fixup-mountpoints

```diff
@@ -467,6 +467,17 @@ case "$DEVICE" in
             "$@"
         ;;
 
+		"shieldtablet")
```

Comment:
Please use tabs everywhere to indent

Reply:
You can fix the change on your computer, then write:
```
git add fixup-mountpoints
git commit --amend
git push --force origin HEAD:master```

Reply:
Please force-push your changes as per instructions here: https://github.com/mer-hybris/hybris-boot/pull/130#discussion_r165023369

Reply:
It should've been force-pushed already, I've ran all the commands you gave me and highlighting the text above (my changes in green) also shows that the indents are tabs.

Reply:
Sorry, I forgot to write in an essential bit: please use spaces instead of tabs everywhere.

I have done this for you, and will add push it to the master myself, so I'll close this PR